src/js/themes/__tests__/__snapshots__/theme-test.js.snap

```diff
@@ -1567,7 +1567,7 @@ exports[`Grommet hpe theme 1`] = `
   outline: none;
   max-width: 100%;
   background: #01A982;
-  color: #FFFFFF;
+  color: #EEEEEE;
```

Comment:
This `color` shouldn't have changed on the theme test, will you be willing to explore why this change occur?

Reply:
Not sure what happened there ... may be I mixed up npm and then yarn in my workflow. We are using 'npm' in other projects, so its quite possible I mixed it up.

I have however reverted these changes and its not included in the final commit.

Reply:
Thanks!